mmdet/models/layers/transformer/utils.py

```diff
@@ -44,6 +44,58 @@ def nchw_to_nlc(x):
     return x.flatten(2).transpose(1, 2).contiguous()
 
 
+def convert_coordinate_to_encoding(pos_tensor: Tensor,
```

Comment:
remember to remove this after conditional and dab merged

Reply:
@jshilong okkkk